### DISCUSSION TITLE
Is there a way to deploy a hybrid of both functions in Astro

### DISCUSSION THREAD

**Discussion by hswmartin** [ddiu8081/chatgpt-demo#278] [General]
I have a question that I would like to ask you. Vercel offers two types of functions: Edge and Serverless. Serverless function has fewer limitations. Is there a way to deploy a hybrid of both functions in Astro?

**Comment by ddiu8081** [comment]
It seems that serverless function does not support streaming output. So the experience is not good when outputting long answers and maybe a timeout when outputting long answers.
